code/modules/mob/living/simple_animal/abnormality/waw/silence.dm

```diff
@@ -40,21 +46,23 @@
 	. = ..()
 	if(meltdown_cooldown < world.time)
 		meltdown_cooldown = world.time + meltdown_cooldown_time
-		sound_to_playing_players('sound/abnormalities/silence/bong.ogg')	//Church bells ringing, whether it happens or not.
 		if(!safe)
 			datum_reference.qliphoth_change(-1)
 		safe = FALSE
 	return
 
 //Meltdown
 /mob/living/simple_animal/hostile/abnormality/silence/zero_qliphoth(mob/living/carbon/human/user)
-	for(var/mob/living/carbon/human/L in GLOB.player_list)
-		if(faction_check_mob(L, FALSE) || L.z != z || L.stat == DEAD)
+	// You have mere seconds to live
+	sound_to_playing_players_on_level('sound/abnormalities/silence/ambience.ogg', 50, zlevel = z)
+	SLEEP_CHECK_DEATH(5 SECONDS)
+	sound_to_playing_players_on_level('sound/abnormalities/silence/price.ogg', 50, zlevel = z)
+	for(var/mob/living/carbon/human/H in GLOB.player_list)
+		if(faction_check_mob(H, FALSE) || H.z != z || H.stat == DEAD)
 			continue
 
-		stoplag(1)
```

Comment:
Originally this was here because I liked the idea of people watching each other get hit by the effect one at a time like a wave of death coming over people as opposed to just hitting everyone at once, though the value here was definitely too low to represent that.

Reply:
It caused issues, as stated in an already open PR by Kirie

Reply:
Damn, sad to hear that.